pyabc/model.py

```diff
@@ -19,10 +19,12 @@ class ModelResult:
     distances and accepted/rejected.
     """
 
-    def __init__(self, sum_stats=None, distance=None, accepted=None):
+    def __init__(self,
+                 sum_stats=None, distance=None, accepted=None, weight=1.0):
```

Comment:
break the arguments in different text lines ?

Reply:
will do and also add type hints, thx